### PR TITLE
[WIP] refactor: new styles for guide

### PR DESCRIPTION
**Pull request by justyna-13**
# Scope of work
Updated styles from Figma

### CODE REVIEW

File: packages/shared/styles/classes/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@mixin color-modifiers($prefix: '-', $property: 'color') {
+  @each $name, $hex in $colors {
+    &#{$prefix}#{$name} {
+      #{$property}: $hex;
+    }
+  }
+}
+
+@mixin space-modifiers($prefix: '-', $property: 'padding') {
```
Comment (filrak) [inline packages/shared/styles/classes/_mixins.scss:9]: isn't that the same as -multiple but with 1 argument? for readability maybe we should have all modifiers as multiple (but named as this one)

File: packages/shared/styles/classes/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  @each $name, $space in $spaces {
+    &#{$prefix}#{$name} {
+      #{$property}: $space !important;
```
Comment (filrak) [inline packages/shared/styles/classes/_mixins.scss:12]: wouldn't that break media queries?

File: packages/shared/styles/classes/_spacing.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
+/* padding */
+.pad {
```
Comment (filrak) [inline packages/shared/styles/classes/_spacing.scss:2]: just `pad` without prefix should use the default spacing (`spacer-base`)

File: packages/shared/styles/variables/_typography.scss [hunk 4]
```diff
@@ -1,36 +1,38 @@
 @import "../helpers";
 :root {
   // font-family
-  --font-family-primary: "Roboto", serif;
-  --font-family-secondary: "Raleway", sans-serif;
+  --font-family--primary: "Roboto", serif;
+  --font-family--secondary: "Raleway", sans-serif;
   // font-weight
   --font-light: 300;
```
Comment (Michal-Dziedzinski) [inline packages/shared/styles/variables/_typography.scss:7]: If the variables responsible for `font-size` are called` --font-size - size`, then maybe the variables responsible for `font-weight` should be written as` --font-weight - weight`, wdyt?

**Comment by AdamPawlinski** [comment]
In my opinion we could add utility class for all screen sizes, for padding and margin. It could be sth like this in mixins.scss: `&#{$name} {
      @for $i from 0 to length($properties) {        
          #{nth($properties, $i + 1)}: $space;
      }
    }`
Wdyt?

File: packages/shared/styles/variables/_typography.scss [hunk 5]
```diff
@@ -1,36 +1,38 @@
...
-  --font-normal: 400;
-  --font-medium: 500;
-  --font-semibold: 600;
-  --font-bold: 700;
-  --font-extra-bold: 800;
-  --font-black: 900;
+  --font-weight--light: 300;
+  --font-weight--normal: 400;
+  --font-weight--medium: 500;
+  --font-weight--semibold: 600;
```
Comment (filrak) [inline packages/shared/styles/variables/_typography.scss:10]: maybe add bold too

File: packages/shared/styles/variables/_typography.scss [hunk 6]
```diff
@@ -1,36 +1,38 @@
...
   // heading font-size
-  --h1-font-size: 1.375rem; //22px
-  --h2-font-size: 1.375rem; //22px
-  --h3-font-size: 1rem; //16px
-  --h4-font-size: 0.875rem; //14px
-  --h5-font-size: 0.75rem; //12px
-  --h6-font-size: 0.75rem; //12px
+  --h1-font-size: 1.5rem; //24px
+  --h2-font-size: 1.5rem; //24px
+  --h3-font-size: 1.125rem; //18px
```
Comment (filrak) [inline packages/shared/styles/variables/_typography.scss:19]: Shouldn't we use `font-size` vars here ?

**Pull request merged**